src/test/ecs/commands/updateEnableExecuteCommandFlag.test.ts

```diff
@@ -33,12 +34,17 @@ describe('updateEnableExecuteCommandFlag', async function () {
 
     it('attempts to enable service', async function () {
         node = new EcsServiceNode(serviceExecDisabled, parent, ecs)
-        sandbox.stub(DefaultSettingsConfiguration.prototype, 'readSetting').returns(false)
```

Comment:
use `TestSettingsConfiguration` (a "fake" implemenation, i.e. something that actually works) instead of a "mock"/"stub" (requires explicit programming in every test)